website/blog/2024-03-21-using-your-private-git-repository-context-for-completion/index.md

```diff
@@ -0,0 +1,113 @@
+---
+authors: [icycodes]
+tags: [deployment, repository context]
+---
+
+# Connect Private GitHub Repository to Tabby
+
+A few months back, we published a blog [Repository context for LLM assisted code completion](https://tabby.tabbyml.com/blog/2023/10/16/repository-context-for-code-completion), introducing the `Repository Context` feature in Tabby.
+
+In this blog, I will guide you through the process of setting up a Tabby server configured with **a private Git repositories** context.
```

Comment:
Consider adding something suggesting that many users have adopted this to incorporate repo-level knowledge into Tabby (to highlight the wide adoption). In order to make this integration process easier and more smooth, we are introducing how to set up Tabby server with private Git repos in this blog.